openslides/core/urls.py

```diff
@@ -16,9 +16,9 @@
         views.VersionView.as_view(),
         name='core_version'),
 
-    url(r'^angular_js/(?P<openslides_app>site|projector)/$',
-        views.AppsJsView.as_view(),
-        name='core_apps_js'),
+    url(r'^openslides_javascript/(?P<realm>site|projector)/$',
```

Comment:
I don't like the ur "openslides_javascript" because it is obvious that openslides give you openslides code, and we could create an alternative client in the future (query mobile?) that is also javascript.

So I prefer somethink with angular and without openslides in it